docs/source/science/ATL08/ATL08.ipynb

```diff
@@ -0,0 +1,1389 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.        with xarray.open_dataset(f, group='gt1l/land_segments', engine="h5netcdf", phony_dims='sort') as atl08_track:
</pre><p>This is a  a slightly odd pattern I normally see people just assign the output of <code>xarray.open_dataset(...</code> to a variable </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/362/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>